build.sh

```diff
@@ -13,4 +13,4 @@ __SDK_PATH="$DOTNET_INSTALL_DIR/sdk/$_ReadGlobalVersion"
 export RepoRoot="$__scriptpath/"
 
 $__DOTNET_CMD $__SDK_PATH/MSBuild.dll $__scriptpath/src/targetPacks/buildTargettingPackages.proj /m /bl:"$__log_path/BuildTargettingPackages.binlog" $@
-$__DOTNET_CMD $__SDK_PATH/MSBuild.dll $__scriptpath/src/referencePackages/buildReferencePackages.proj /bl:"$__log_path/BuildReferencePackages.binlog" $@
+# $__DOTNET_CMD $__SDK_PATH/MSBuild.dll $__scriptpath/src/referencePackages/buildReferencePackages.proj /bl:"$__log_path/BuildReferencePackages.binlog" $@
```

Comment:
NIT: Isn't this needed anymore? Maybe you can delete it.

Reply:
This is still needed. I commented a part of build to reduce the build time and test the changes. Testing - WIP.